### PR TITLE
[PCL] Fixes VTK 7 support for 1.7.2

### PR DESCRIPTION
**Pull request by fran6co**
This fixes 2 issues:

  * ```brew install pcl``` isn't using the bottle because VTK is built without qt support by default
  * 1.7.2 doesn't support VTK 7 (https://github.com/Homebrew/homebrew-science/pull/3512, https://github.com/Homebrew/homebrew-science/issues/3370, https://github.com/Homebrew/homebrew-science/issues/3347)

Merges:

  * https://github.com/PointCloudLibrary/pcl/pull/1500
  * https://github.com/PointCloudLibrary/pcl/pull/1534

### CODE REVIEW

File: pcl.rb [hunk 1]
```diff
@@ -31,19 +31,30 @@ def message
...
+  revision 1
 
   stable do
+    url "https://github.com/PointCloudLibrary/pcl/archive/pcl-1.7.2.tar.gz"
+    sha256 "479f84f2c658a6319b78271111251b4c2d6cf07643421b66bbc351d9bed0ae93"
     patch do
       url "https://gist.githubusercontent.com/fran6co/a6e1e44b1b43b2d150cd/raw/0c4aeb301ed523c81cd57c63b0a9804d49af9848/boost.patch"
       sha256 "5409b0899f65d918248a8fdfb820478cc0b191c50339e16692a911fab76c3f43"
     end
     # Fixes PCL for VTK 6.2.0
```
Comment (dpo) [inline pcl.rb]: Is this useful? We don't support VTK 6.

Comment (fran6co) [inline pcl.rb]: Yes, it's actually a patch for VTK >= 6.2.0

Comment (dpo) [inline pcl.rb]: There's already a patch below for VTK 7, which is the only one in homebrew/science. I'm just checking whether both patches are needed for VTK 7.

Comment (fran6co) [inline pcl.rb]: The patch below builds on the previous patch. This is going to go all away when 1.8.0 gets released. 1.7.2 is fairly old already.

Comment (dpo) [inline pcl.rb]: Ok, thank you for clarifying.

**Pull request closed without merging**